src/pf-tooltip/index.html

```diff
@@ -0,0 +1,51 @@
+<!doctype html>
+<html>
+<head>
+    <title>pf-tooltip example</title>
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/patternfly/3.13.0/css/patternfly-additions.css">
+    <link rel="stylesheet" href="/dist/css/patternfly.css">
+    <script src="//cdnjs.cloudflare.com/ajax/libs/webcomponentsjs/0.7.22/webcomponents.js"></script>
+    <script src="/dist/js/pf-utils.js"></script>
+    <script src="/dist/js/pf-tooltip.js"></script>
+</head>
+<body>
+<div class="container">
+    <div class="page-header">
+        <h1>Tooltips</h1>
+    </div>
+    <hr>
+
+    <pf-tooltip placement="left" targetSelector="#btn-left">
+        Tooltip text on Left
+    </pf-tooltip>
+    <button id="btn-left" type="button" class="btn btn-primary">Left</button>
+
+    <pf-tooltip placement="top" targetSelector="#btn-top">
+        <span>Tooltip on top</span>
+    </pf-tooltip>
+    <button id="btn-top" type="button" class="btn btn-secondary">Top</button>
+
+    <pf-tooltip placement="bottom" targetSelector="#btn-bottom">
+        <span>Tooltip on bottom</span>
+    </pf-tooltip>
+    <button id="btn-bottom" type="button" class="btn btn-warning">Bottom</button>
+
+    <pf-tooltip placement="right" targetSelector="#btn-right">
+        <span>Tooltip on right</span>
+    </pf-tooltip>
+    <button id="btn-right" type="button" class="btn btn-danger">Right</button>
+
+    <br/>
+    <br/>
+    <script>
+        setTimeout(function(){
+          var pfTooltipRight = document.querySelector('[targetSelector="#btn-right"]');
+          pfTooltipRight.innerHTML = "Right content changed!";
+
+          var event = new CustomEvent('handleContentChanged',{});
+          pfTooltipRight.dispatchEvent(event);
```

Comment:
Per conversation today, i'll add a method to call the event (as opposed to requiring consumers to fire the event). This should be a more consumable API to those consuming the component directly (without a shim). This approach leveraged with the Tooltip should be reusable in other components.